package.json

```diff
@@ -8,7 +8,8 @@
     "serve": "webpack-dev-server --watch --progress",
     "serve:prod": "webpack-dev-server --env=prod",
     "build:dev": "webpack",
-    "build:prod": "webpack --env=prod"
+    "build:prod": "webpack --env=prod",
+    "postinstall": "cd ./node_modules/esn-frontend-login && npm install && npm run build"
```

Comment:
This step is mandatory since things are defined in devDependencies in Vue apps.
If we do not want to do such thing, we need to install and run the Vue app with `npm run serve` then proxy it in the current SPA with the help of webpack-dev-server.